js/dashboard.js

```diff
@@ -199,6 +199,110 @@ $(function() {
         }
     });
 
+    $('#leafwetness-button').click(function() {
+				$.ajax({
+            url: 'http://localhost:8000/env_data?vineyard_id=0&env_variable=leafwetness',
+            type: "GET"
+        }).done(function(json) {
+            if (json.env_data) {
+                window.location.href = "http://localhost:8000/env_data?vineyard_id=0&env_variable=leafwetness";
+            } else {
+                // go nowhere
```

Comment:
Okay, after coffee, I'm thinking more clearly. I added those as basic placeholders. Didn't we say in the design phase that we would error handle? Depending on when the error occurs, shouldn't we have it go to a basic "here's the thing that went wrong, sorry. Try to reload the page in a few minutes" page?